ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -34,6 +43,12 @@ const styles = StyleSheet.create({
   },
   alignedRight: {
     textAlign: "right"
+  },
+  alignedLeft: {
+    textAlign: "left"
+  },
+  H3: {
+    fontSize: variables.fontSizeBase * 1.25
```

Comment:
isn't this the current behavior of H3? (./theme/components/H3.ts)

Reply:
yes, I missed it during the last review